internal/handler/msg_usersinfo.go

```diff
@@ -169,6 +169,18 @@ func (h *Handler) MsgUsersInfo(ctx context.Context, msg *wire.OpMsg) (*wire.OpMs
 			return nil, lazyerrors.Error(err)
 		}
 
+		if v.Has("credentials") {
+			credentials := must.NotFail(v.Get("credentials")).(*types.Document)
+			if credentialsKeys := credentials.Keys(); len(credentialsKeys) > 0 {
+				mechanisms := must.NotFail(types.NewArray())
+				for _, k := range credentialsKeys {
```

Comment:
If `credentialsKeys` is a slice of strings, I guess we can do something like `types.NewArray(credentialsKeys...)` to save a few lines of code :)

Reply:
I'm afraid we can't do that because types.NewArray receives a slice of []any and credentialsKeys is a slice of []string.